### ISSUE TITLE
Add descriptions to graphs in the documentation

### ISSUE THREAD

**Issue by dqnykamp** [Doenet/DoenetML#752]
For accessibility, each `<graph>` should have its `description` attribute set to a description of the graph. Each graph in the documentation need a description written.

Below is a list of document sections with graphs in need of a description. Ask whoever merges in your PR to check off the sections you completed.

### Reference
- [x]  essentialComponents
- [x]  abs
- [x] and
- [x] angle
- [ ] animateFromSequence
- [ ] answer
- [ ] attractTo 
- [ ] attractToGrid
- [x] award
- [ ] bestFitLine
- [ ] bezierControls
- [x] boolean
- [x] booleanInput
- [ ] callAction
- [x] choiceInput
- [ ] circle
- [ ] collect
- [x] conditionalContent
- [x] considerAsResponses
- [x] constrainTo
- [ ] constrainToGraph
- [ ] contrainToGrid
- [ ] contraintUnion
- [ ] controlVectors
- [x] coords
- [x] count
- [ ] curve
- [ ] derivative
- [x] div
- [ ] endpoint
- [ ] function
- [x] graph
- [x] group
- [ ] image
- [ ] intersection
- [x] interval
- [x] intervalList
- [ ] label
- [ ] legend
- [x] li
- [ ] line
- [ ] lineSegment
- [ ] math
- [x] mathInput
- [x] matrix
- [x] matrixInput
- [x] md
- [x] module
- [x] moduleAttributes
- [ ] number
- [ ] odeSystem
- [x] ol
- [x] option
- [x] or
- [x] p
- [x] paginator
- [ ] parabola
- [ ] piecewiseFunction
- [ ] point
- [ ] polygon
- [ ] polyline
- [x] problem
- [ ] ray
- [ ] rectangle
- [ ] regionBetweenCurves
- [ ] regionBetweenCurvesXAxis
- [ ] regularPolygon
- [x] repeat
- [x] repeatForSequence
- [ ] sbsGroup
- [x] select
- [x] sequence
- [ ] setup
- [ ] sideBySide
- [x] sign
- [ ] solution
- [ ] sort
- [x] stack
- [ ] textInput
- [ ] triangle
- [ ] triggerSet
- [x] ul
- [ ] updateValue
- [ ] vector
- [x] when
- [ ] xLabel
- [x] xor
- [ ] yLabel

### Sandbox
- [x] puzzles
- [ ] solutions

### Tutorials
- [ ] intermediateTutorial
- [x] googleSite
